test/fuzz/env.ml

```diff
@@ -1,20 +1,22 @@
 open Owi.Symbolic
 open Crowbar
 
+type block_kind =
+  | Block
+  | Loop
```

Comment:
You should add `Func` and put only one of them when you start generating a function.